resources/definitions/ultimaker_s7.def.json

```diff
@@ -9,9 +9,7 @@
         "manufacturer": "Ultimaker B.V.",
         "file_formats": "application/x-ufp;text/x-gcode",
         "platform": "ultimaker_s7_platform.obj",
-        "bom_numbers": [
-            5078167
-        ],
+        "bom_numbers": [5078167, 5078168],
         "firmware_update_info":
         {
             "check_urls": [ "https://software.ultimaker.com/releases/firmware/9051/stable/um-update.swu.version" ],
```

Comment:
The `check_urls` for the S7 is incorrect, no? It's pointing to the`9051` directory on GCP.

Reply:
Seems its been incorrect since this definition was created, might as well correct that and see if Cura agree to include it.